easybuild/easyblocks/f/fftw.py

```diff
@@ -122,6 +122,9 @@ def __init__(self, *args, **kwargs):
             # on macOS, AVX is indicated with 'avx1.0' rather than 'avx'
             if 'avx1.0' in avail_cpu_features:
                 avail_cpu_features.append('avx')
+            # avx512 availability is indicated via avx512f and other features starting with avx512
+            if 'avx512f' in avail_cpu_features:
```

Comment:
@bartoldeman Maybe this is a better idea?

```python
if any(feat.startswith('avx512') for feat in avail_cpu_features):
    avail_cpu_features.append('avx512')

Reply:
the reason is here:
https://github.com/FFTW/fftw3/blob/master/simd-support/simd-avx512.h#L47
The avx512 support in FFTW uses the common subset of Skylake X avx512 and Knights Landing avx512, which is avx512f (foundation, lots of instructions) and avx512cd (conflict detection, just three instructions), but of those two only avx512f is used.

Reply:
@bartoldeman OK, thanks for clarifying. Maybe we should include a comment in the code itself to clarify this?